python/halo/odla.py

```diff
@@ -83,5 +86,8 @@ def Execute(self, data):
             buf = (c_float * v[2])()  # FIXME: handle types
             self.h.odla_BindToOutput(v[0], buf, self.ctx)
             buffers.append(buf)
+        s = time()
         self.h.odla_ExecuteComputation(self.comp, self.ctx, 0, c_void_p(0))
+        t = time()
+        self.logger.info("Execution time:" + str(t - s))
```

Comment:
maybe we can add a unit (sec, milisec, etc.)

Reply:
OK